CHANGELOG.asciidoc

```diff
@@ -24,7 +24,7 @@ image::https://raw.githubusercontent.com/apache/tinkerpop/master/docs/static/ima
 === TinkerPop 3.3.6 (Release Date: NOT OFFICIALLY RELEASED YET)
 
 * Masked sensitive configuration options in the logs of `KryoShimServiceLoader`.
-* Fixed a concurrency issue in `TraverserSet`
+* Fixed concurrency issues in `TraverserSet::toString()` and `ObjectWritable::toString()`.
```

Comment:
we tend to just use the "." rather than the "::" when referencing methods on specific classes.

Reply:
Ha, you're picky. "::" is the official notation for method references though.

Reply:
sorry - just trying to be consistent.